src/util/html-join.ts

```diff
@@ -0,0 +1,11 @@
+import type { TemplateResult } from "lit";
+import { html, nothing } from "lit";
+
+export const htmlJoin = (
```

Comment:
Unused?

Reply:
Yeah, I did that before I realized lit has a built it directive for that. I removed it